x-pack/legacy/plugins/monitoring/public/views/elasticsearch/index/index.js

```diff
@@ -100,6 +100,7 @@ uiRoutes.when('/elasticsearch/indices/:index', {
               onBrush={this.onBrush}
               indexUuid={this.indexName}
               clusterUuid={$scope.cluster.cluster_uuid}
+              zoomInfo={this.zoomInfo}
```

Comment:
💯 Even though this is "The React Way" of doing things, I think this would have been handled better in a state manager somewhere.

Another way is to pass **one** object that holds all of the props. Works really well with typescript and interfaces